{"uuid":"b73c7eb2-b793-47dc-87f9-b9531905f431","name":"Default test","children":["1cf73442-6f4c-4893-8a35-191bb579207a","8502025f-2cd5-4269-b45e-d074e48f9bce","0e270268-c818-4b27-b5ac-bc285db211c4","3d24975b-ec6a-4fae-8a5c-895ae1ddfba7","5802212f-095f-49d4-ac64-0a28e27f474a","bbe7cb30-4fc6-4a4e-b27d-a69917b8f1ae","d8645ab8-7d94-4b37-bc01-9ef0e4a0a88f","0d6dee62-ed4a-448d-9fd8-9f00317a8c46"],"befores":[{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver","trace":"org.openqa.selenium.remote.NoSuchDriverException: chromedriver located at path/to/chromedriver.exe, but invalid\nFor documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/\nBuild info: version: '4.12.1', revision: '8e34639b11'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.8.1'\nDriver info: driver.version: ChromeDriver\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:45)\r\n\tat org.openqa.selenium.remote.service.DriverFinder.getPath(DriverFinder.java:13)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.generateExecutor(ChromeDriver.java:99)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:88)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:83)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:51)\r\n\tat runner.FeatureRunner.setUp(FeatureRunner.java:21)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:663)\r\n\tat org.testng.TestRunner.beforeRun(TestRunner.java:652)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699666891699,"stop":1699666891917}],"afters":[{"name":"tearDown","status":"broken","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1699666893079,"stop":1699666893079}],"start":1699666891651,"stop":1699666893085}